tests/e2e-pw/playwright.config.ts

```diff
@@ -67,6 +67,10 @@ const config: ExtendedPlaywrightTestConfig = {
 			testMatch: /.*.classic_theme.spec.ts/,
 			dependencies: [ 'classicThemeConfiguration' ],
 		},
+		{
+			name: 'cartCheckout',
+			testMatch: /cart-block.*/,
+		},
```

Comment:
Why did you add a new configuration?

Reply:
If we want to split out the FSE tests and Cart/Checkout tests (I did not add the `checkout-block` directory because it doesn't exist yet) this is the best way to achieve that?

Reply:
Do you think we should do it?

Reply:
I think so, during the Woo DM we had a team time meeting, and we spoke about splitting the cart/checkout tests and the FSE tests, most of the team were in favour of that, cc @ralucaStan do you still support this?

Reply:
I think this is valuable also, as we are 2 different teams, we may want to run just cart & checkout tests and you guys might want to run the FSE tests. If not in CI, then definitely locally for speed of development. What might get tricky here is the permutations. For example, I notice there are `blockTheme` and `classicTheme` projects. What happens if we want to test the cart checkout for both classic and block themes? Would the tests go in the `cartCheckout` project or one of the other two mentioned above?